notebooks/Tutorial10_Geolocation_privacy_assessment.py

```diff
@@ -0,0 +1,251 @@
+noised_dataset = client.pandas_integration.upload_dataframe(noised_df)
+print(f"Dataset {noised_dataset.id} loaded")
+
+# ### Create scenarios
+#
+# A privacy assessment for geolocation data requires the definition of potentially multiple scenarios.
+# A scenario defines some attacks that will be considered during the assessment. One scenario contains the following parameters:
+# - `projection_parameters` defines the method of projection (or representation) of the data that will be used to evaluate the level of protection against **singling-out attacks**. For example, an attack can attempt re-identification using trace density information (this projection is set by using `GeolocationDensityParameters`). An alternative is for re-identification of traces to be performed from calculated features such as mean speed, trip duration ... This projection can be chosen by setting the parameters with `GeolocationFeaturesParameters`.
+# - `known_features` defines a calculated feature that may be known to an attacker and that could be used to link the anonymized data with data from an external source. This attack is a **feature-based linkability attack**.
+# - `target_feature` defines a calculated feature that could be considered as sensitive and should be protected against **feature-based inference attacks**. Such attack would use the `known_features` to infer the `target_feature`.
+# - `known_signal_position` defines a part of the trip that could be known to an attacker (i.e. the start, middle or end of a trip). Attempt to re-associate anonymized trips to their original counterparts only using this information is considered as a **trace-based linkability attack**.
+# - `target_signal_poi` defines the point of interest (POI) that could be considered as a target for an attacker in the context of a **trace-based inference attack**. It can be the start or the end of a trip.
+# - `inference_metric_threshold` defines the distance under which a prediction on a target point of interest will be considered as a successful prediction by the attacker. Setting a large threshold will increase the expected level of protection.
+#
+# The ability to define many scenarios ensure that the data can be looked from different angles and that the privacy assessment cover many types of attacks. There is no limit on the number of scenarios that can be defined.
+#
+# Note that only `projection_parameters` must be set in a scenario, the remaining fields are optional.
+#
+# The example below defines 3 scenarios.
+
+scenarios = [
+    # Scenario 0
+    PrivacyMetricsGeolocationScenario(
+        projection_parameters=GeolocationDensityParameters(),
+        known_features=[
+            GeolocationFeatures.duration,
+            GeolocationFeatures.length,
+        ],
+        target_feature=GeolocationFeatures.speed,
+        known_signal_position=SignalPosition.start,
+        target_signal_poi=PointOfInterest.end,
+        inference_metric_threshold=20000,
```

Comment:
```suggestion
        inference_metric_threshold=1000, # Distance in meters
```
Looks more reasonable (I set 20000 in the geolocation test to see if the metric works)